src/components/RightSidebar/MoreMenu.astro

```diff
@@ -58,6 +58,25 @@ const { editHref } = Astro.props;
 			<span>Join our community</span>
 		</a>
 	</li>
+	<li class={`header-link depth-2`}>
+		<a href="https://github.com/withastro/astro" target="_blank">
```

Comment:
Oops, just noticed: this url should go to https://github.com/withastro/docs to go to the repository for this documentation, not the Astro project as a whole.

Reply:
Ah I figured someone coming to the docs might want to go to the source code for the project. I can update this.

Reply:
Yeah, I second guessed myself, too, @chriswburke, but looking back at @Princesseuh 's original issue, she had wanted a way to get to the docs repository itself quickly. So I think this change satisfies her original request!

https://github.com/withastro/docs/issues/176